hbase-it/pom.xml

```diff
@@ -375,6 +375,7 @@
         <dependency>
           <groupId>org.apache.hadoop</groupId>
           <artifactId>hadoop-minicluster</artifactId>
+          <scope>test</scope>
```

Comment:
I think we should leve these as compile scope.

hbase-it is dependend upon by several external projects, and having these in compile scope ensures that they will get the transitive dependencies without having to depend on them explicitly.

Now I'm not sure that depending on hbase-it is actually a good idea, external projects should probably depend on hbase-testing-util, and any other necessary hbase module explicitly, but even Phoenix depends on hbase-it.

Reply:
Or we could just leave it like this, since they are going to have to make changes for HBase 3.0 anyway.

Reply:
and force them to fix their test dependencies.

Reply:
Marking as resolved!